### PR TITLE
Add com.discordapp.DiscordPTB

### PR DESCRIPTION
**Pull request by kuflierl**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*    **Access to camera is highly recommended for discord. Filesystem ro access to Videos and Pictures is required for selecting files. Filesystem rw access is required for downloading files**
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type. **I assume it is ok since [com.discordapp.Discord](https://github.com/flathub/com.discordapp.Discord) exists**
- [ ] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [ ] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)* **The updater is not compatible with flatpak**

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

see #3580

### CODE REVIEW

**Comment by barthalion** [comment]
How is this different from Discord Canary in flathub-beta?

**Comment by kuflierl** [comment]
> How is this different from Discord Canary in flathub-beta?

see https://github.com/flathub/flathub/pull/3580#issuecomment-1276514184

File: com.discordapp.DiscordPTB.appdata.xml [hunk 1]
```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<application>
+  <id type="desktop">com.discordapp.DiscordPTB.desktop</id>
```
Comment (hfiguiere) [inline com.discordapp.DiscordPTB.appdata.xml:3]:
```suggestion
  <id type="desktop">com.discordapp.DiscordPTB</id>
```

never `.desktop`

File: com.discordapp.DiscordPTB.json [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+				{
+					"type": "file",
+					"path": "Twemoji.ttf"
+				}
+			],
+			"build-commands": [
+				"mkdir -p /app/share/fonts/",
+				"cp Twemoji.ttf /app/share/fonts/",
+				"mkdir -p /app/etc/fonts/",
+				"cp emoji-font.conf /app/etc/fonts/local.conf",
```
Comment (hfiguiere) [inline com.discordapp.DiscordPTB.json:44]:
```suggestion
				"install -Dm644 Twemoji.ttf -t /app/share/fonts/",
				"install -Dm644 emoji-font.conf /app/etc/fonts/local.conf",
```

File: com.discordapp.DiscordPTB.json [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+			"buildsystem": "simple",
+			"build-commands": [
+				"install -Dm755 discord-ptb.sh /app/bin/discord-ptb",
+				"install -Dm755 set-gtk-dark-theme.py /app/bin",
+				"install -Dm755 disable-breaking-updates.py /app/bin",
+				"mv DiscordPTB /app/discord-ptb",
+				"chmod +x /app/discord-ptb/DiscordPTB",
+				"install -d /app/share/applications",
+				"desktop-file-edit --set-icon=\"com.discordapp.DiscordPTB\" --set-key=\"Exec\" --set-value=\"discord-ptb\" /app/discord-ptb/discord-ptb.desktop",
+				"cp /app/discord-ptb/discord-ptb.desktop /app/share/applications/${FLATPAK_ID}.desktop",
```
Comment (hfiguiere) [inline com.discordapp.DiscordPTB.json:100]:
```suggestion
				"desktop-file-edit --set-icon=\"com.discordapp.DiscordPTB\" --set-key=\"Exec\" --set-value=\"discord-ptb\" /app/discord-ptb/discord-ptb.desktop",
				"install -Dm644 /app/discord-ptb/discord-ptb.desktop /app/share/applications/${FLATPAK_ID}.desktop",
```

**Review by hfiguiere — CHANGES_REQUESTED** [review]

**Comment by kuflierl** [comment]
bot, build com.discordapp.DiscordPTB

**Comment by kuflierl** [comment]
Quick question, how long does it generally take for requests to be accepted/denied?

**Comment by barthalion** [comment]
We generally don't outright reject submissions, but given this is yet another non-upstream extra-data submission, for an alpha version of the app, I'm not particularly convinced about how useful it is. I guess we could publish it to Flathub Beta, but I don't think it will end up widely used.

**Comment by kuflierl** [comment]
sooooooo, this pr is kinda dead

File: discord-ptb.sh [hunk 4]
```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+FLATPAK_ID=${FLATPAK_ID:-"com.discordapp.DiscordPTB"}
+# RPC was discontinued in DiscordPTB
```
Comment (Hacksore) [inline discord-ptb.sh:3]: Out of curiosity is there somewhere that this was publicized? I've been using the RPC protocol on IPC and Websockets and would like to know if/when discord plans to fully kill it.

Thanks in advanced!

Comment (kuflierl) [reply discord-ptb.sh:3]: Honestly, this was something I did a while ago so I don't remember from where I got the info (probably some forum) but if I remember correctly, ptb didn't seem to Open rpc (even though I tried) and I also https://discord.com/developers/docs/rich-presence/best-practices deprecates the standard

**Pull request closed without merging**